3drepo/3drepo.io
# ISSUE #3919 - Confirm modal appears when removing an admin user from teamspace

This fixes #3919 
#### Description
- In V5 if you try to remove a teamspace admin from a teamspace you get a confirmation modal
- There are no modals for other permissions


#### Test cases
- Try to remove a teamspace admin
- Try to remove a non-teamspace admin
- Check V4 is unaffected (if a user is a collaborator on a model there will be a modal)
